alonerpg/views.py

```diff
@@ -23,6 +23,14 @@ def get(self, request, *args, **kwargs):
             command = "lt " + request.GET["searchfield"]
             filtered_tables = dynamic_call(self.tables, command)
             if isinstance(filtered_tables.data, list):
+                if "favourites_checkbox" in request.GET:
+                    #if request.GET["favourites_checkbox"] != "":
+                        #filtered_tables.data = get_tables_by_list(filtered_tables.data, request.GET["favourites_checkbox"].split(","))
+                    favourites_checkbox = request.GET["favourites_checkbox"] 
+                    if favourites_checkbox == "":
+                        filtered_tables.data = []
+                    else:
+                        filtered_tables.data = get_tables_by_list(filtered_tables.data, request.GET["favourites_checkbox"].split(","))
```

Comment:
Here you can use `favourites_checkbox`  instead of writing `equest.GET["favourites_checkbox"]` again

Reply:
Done